creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -497,6 +540,7 @@ const _issueUpdateReplicaSetOp = async (
       // If for some reason any node in the new replica set is not registered on chain as a valid SP and is
```

Comment:
i think line 536 above should have its own return value

Reply:
I don't think I follow?

Reply:
line 536: `if (!issueReconfig) return response`
my comment was suggesting that instead of returning `success` here we return a more specific value